power_of_three

```diff
@@ -0,0 +1,17 @@
+def is_power_of_three(n)
+  n/=3 while n>1&&n%3==0
+  n==1?true:false
+end
+
+a=1
+b=3
+c=9
+d=27
+e=77
+f=243
+p "in:#{a} , out:#{is_power_of_three(a)}"
```

Comment:
you can test your method by 
is_power_of_three(3)
is_power_of_three(4)==false

.

Avoid to use unnecessary variables.